app/lib/service/download_counts/computations.dart

```diff
@@ -42,3 +42,29 @@ Future<void> upload30DaysTotal(Map<String, int> counts) async {
   await uploadBytesWithRetry(reportsBucket, downloadCounts30DaysTotalsFileName,
       jsonUtf8Encoder.convert(counts));
 }
+
+Future<({List<int> weeklyPoints, DateTime? newestDate})> computeWeeklyDownloads(
```

Comment:
```suggestion
/// Compute [weeklyDownloads] starting from [newestDate].
///
/// Each number in [weeklyDownloads] is the number of downloads for
/// a given 7 day period. Starting from [newestDate],
Future<({List<int> weeklyDownloads, DateTime? newestDate})> computeWeeklyDownloads(
```

nit: I'd consider avoid using `Points` to avoid confusing with pub points.

Reply:
Updated comment.